packages/liveblocks-zustand/types/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2015",
     "module": "es2020",
     "moduleResolution": "node",
-    "lib": ["es2015"],
+    "lib": ["es2015", "DOM"],
```

Comment:
Why is it needed?

Reply:
This `tsconfig` is only needed for `dtslint` validation. Because we're using fetch in `@liveblocks/client`, this is needed.